### DISCUSSION TITLE
Get the list of documents affected after doing hierarchical_topics method

### DISCUSSION THREAD

**Discussion by stanley-tarce** [MaartenGr/BERTopic#2151] [Q&A] [answered]
Basically, I want to determine the list of documents affected for each left child topic, right child topic, and parent topic (if it has documents listed) after doing this function. 

 hierarchical_topics = topic_model.hierarchical_topics(docs=docs)

Any idea how can I get the list of documents for hierarchical topics? Thank you!

**Comment by MaartenGr** [comment, marked as answer]
You would have to extract those manually since you have topic assignments with `topic_model.topics_`. Using those together with `hierarchical_topics` you can check which documents might end up merged with documents in another topic.

**Reply by stanley-tarce** [reply]
Oh okay! So that means, when you do the hierarchical topics method, it won't alter the topic assignment from the response of the `topic_model.fit_and_transform` method?

**Reply by MaartenGr** [reply]
That's correct!
